migrations/20171211100317_add_created_by_to_questionnaires.js

```diff
@@ -0,0 +1,17 @@
+const TABLE = "Questionnaires";
+const COLUMN = "createdBy";
+
+exports.up = function(knex) {
+  return knex.schema.table(TABLE, table => {
+    table
+      .string(COLUMN)
+      .defaultsTo("")
```

Comment:
Should this default to "Guest"? If we don't set this at migration time, you end up with created by `Unknown` in the questionnaire list. Not a huge deal at this point, so I'll leave it up to you to decide if it makes sense to default or not.